LBH.AdultSocialCare.Api/V1/Controllers/ResidentialCare/ResidentialCarePackageController.cs

```diff
@@ -1,16 +1,18 @@
 using System;
+using System.Collections.Generic;
 using System.Threading.Tasks;
 using LBH.AdultSocialCare.Api.V1.Boundary.Request;
 using LBH.AdultSocialCare.Api.V1.Boundary.Response;
 using LBH.AdultSocialCare.Api.V1.Domain;
 using LBH.AdultSocialCare.Api.V1.Factories;
+using LBH.AdultSocialCare.Api.V1.Infrastructure.Entities;
 using LBH.AdultSocialCare.Api.V1.UseCase.Interfaces;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 
-namespace LBH.AdultSocialCare.Api.V1.Controllers
+namespace LBH.AdultSocialCare.Api.V1.Controllers.ResidentialCare
 {
-    [Route("api/v1/residential-care-package")]
+    [Route("api/v1/[controller]")]
```

Comment:
I think it is better if we dictate the controller URL instead of `[controller]`. Gives flexibility to change controller name in future without affecting FE